test/e2e/suite/command/policy.go

```diff
@@ -60,7 +70,9 @@ var _ = Describe("trust policy maintainer", func() {
 		It("should fail if no file path is provided", func() {
```

Comment:
Can we add another test case with more than one argument? For example, two trust policy files are provided.

Reply:
Updated.